app/views/handlers/individual_response.py

```diff
@@ -147,6 +151,26 @@ def _publish_fulfilment_request(self, mobile_number=None):
             topic_id, message=fulfilment_request.payload
         )
 
+    def _check_individual_response_count(self):
+        if (
+            self._questionnaire_store.response_metadata.get(
+                "individual_response_count", 0
+            )
+            >= current_app.config["EQ_INDIVIDUAL_RESPONSE_LIMIT"]
+        ):
+            raise IndividualResponseLimitExceeded(
+                "Individual response limit has been exceeded"
+            )
+
+    def _update_individual_response_count(self):
+        response_metadata = self._questionnaire_store.response_metadata
+
+        if response_metadata.get("individual_response_count"):
+            response_metadata["individual_response_count"] += 1
+        else:
+            response_metadata["individual_response_count"] = 1
+        self._questionnaire_store.save()
```

Comment:
Calling save from this method will lead to a double save, as it's also called from the `_update_section_status` method. Saving should be done independently of both (and only if the questionnaire store has been updated).

Reply:
Good spot, but do we need to check the store is dirty? If it gets past the count check, the count will always go up by 
 one, so it will always be dirty

Reply:
Ah, something else uses it, I will make a method for just saving questionnaire store with a dirty check